x-pack/package.json

```diff
@@ -46,27 +47,44 @@
     "@types/d3-time": "^1.0.7",
     "@types/d3-time-format": "^2.1.0",
     "@types/elasticsearch": "^5.0.30",
+    "@types/file-type": "^5.2.1",
+    "@types/get-port": "^3.2.0 ",
+    "@types/git-url-parse": "^9.0.0",
+    "@types/glob": "^5.0.35",
     "@types/graphql": "^0.13.1",
     "@types/history": "^4.6.2",
     "@types/jest": "^24.0.9",
     "@types/joi": "^13.4.2",
+    "@types/js-yaml": "^3.11.1",
     "@types/json-stable-stringify": "^1.0.32",
     "@types/jsonwebtoken": "^7.2.7",
     "@types/lodash": "^3.10.1",
+    "@types/mkdirp": "^0.5.2",
     "@types/mocha": "^5.2.6",
+    "@types/nock": "^9.3.0",
```

Comment:
We should include a dependency for `nock`

Reply:
should we add them even they are already in root/package.json?

Reply:
Yep